backend/app/api_cheese.py

```
@@ -99,184 +102,199 @@ def setup_cheese_user_task(app, user_id):
+                        room = session.query(TrackedRoom).filter_by(room_id=ap_room_id).first()
+                        if room and not room.cheese_tracker_id:
+                            room.cheese_tracker_id = ct_id
+                            logging.info(f"[CHEESE_DEBUG] Linked existing AP room {ap_room_id} to Cheese ID {ct_id}")
+
+                if not room:
+                    ap_room_id_extracted = extract_ap_room_id(room_link) or "PENDING_DISCOVERY_" + ct_id[:8]
+                    hostname = "archipelago.gg"
+                    try:
+                        if room_link:
+                            hostname = urlparse(room_link).hostname or "archipelago.gg"
+                    except: pass
```

Ruff (0.15.18)

[error] Do not use bare `except`

(E722)